ng/wasm/Cargo.toml

```diff
@@ -0,0 +1,82 @@
+[package]
+name = "wasm"
+version = "0.1.0"
+edition = "2018"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+
+[dependencies]
+wasm-bindgen = { version = "0.2", features = ["serde-serialize"] }
+blsful = { version = "2", default-features = false, features = ["rust"] }
+base64_light = "0.1"
+getrandom = { version = "0.2", features = ["js"] }
+hex = "0.4"
+serde = "1.0"
+serde_json = "1.0"
+serde_bare = "0.5"
+serde-wasm-bindgen = "0.5"
+bls12_381_plus = { version = "=0.8.9" }
```

Comment:
Not needed since you're including blsful. Use blsful::inner_types and it gets you the same.

Reply:
Agree.

I added this to fix the crate to an older version, which would fix a compile error. IIRC, I got the error when using the `blsful` crate from Git, but it doesn't occur at all using versions published to crates.io. Removing.